packages/main/test/pages/Kitchen.html

```diff
@@ -9,6 +9,12 @@
 
 	<link rel="stylesheet" type="text/css" href="./kitchen-styles.css">
 
+    <style>
+        *:not(:defined) {
+            visibility: hidden;
```

Comment:
why not display:none ? if this case is valid, let's describe it as well in the FAQ.